.pre-commit-config.yaml

```
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.7.4
+    hooks:
+      - id: pyupgrade
+        args: ["--py3-plus"]
```

it might also be worth running with `--py36-plus` and dropping support for EOL py3.5

```suggestion
        args: ["--py36-plus"]
```